docs/avs/service-components.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 3
+sidebar_label: "🧩 Service Components"
+---
+
+# Service Components
+
+The AVS operates through several key components, each playing a crucial role in the validation and aggregation process.
+
+## Task Performer
+
+The Task Performer is responsible for fetching data from multiple oracle nodes using a gateway.
```

Comment:
You could use DDL (data delivery layer) to use the same nomenclature as in other docs.